src/System.IO.FileSystem/src/System/IO/FileSystemInfo.Unix.cs

```diff
@@ -163,9 +163,10 @@ DateTimeOffset IFileSystemObject.CreationTime
             get
             {
                 EnsureStatInitialized();
-                return (_fileStatus.Flags & Interop.Sys.FileStatusFlags.HasBirthTime) != 0 ?
-                    DateTimeOffset.FromUnixTimeSeconds(_fileStatus.BirthTime).ToLocalTime() :
-                    default(DateTimeOffset);
+                long rawTime = (_fileStatus.Flags & Interop.Sys.FileStatusFlags.HasBirthTime) != 0 ?
+                    _fileStatus.BirthTime :
+                    Math.Min(_fileStatus.ATime, _fileStatus.CTime); // fall back to the older of last access / status change time
```

Comment:
Only if that's what the OS gave back for st_atime / st_ctime.  We just stat and then copy out the values.

The only thing I went back and forth on here was whether to use CTime vs MTime (last write time).  We don't expose CTime anywhere, which makes this slightly harder to explain (it'd be easy to say we just fall back to the lower of LastAccessTime / LastWriteTime).  But CTime is what Mono uses, and in practice it really shouldn't matter much.